packages/node/package.json

```
@@ -65,7 +65,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@fastify/otel": "0.6.0",
+    "@fastify/otel": "git+https://github.com/getsentry/fastify-otel.git#otel-v1",
```

```suggestion
    "@fastify/otel": "getsentry/fastify-otel#otel-v1",
```

Not sure if this is really much different, but I've seen this syntax more often?